runtime/darwinia/src/pallets/fee_market.rs

```diff
@@ -0,0 +1,57 @@
+// --- core ---
+use core::cmp;
+// --- substrate ---
+use frame_support::{traits::LockIdentifier, PalletId};
+use sp_runtime::{traits::UniqueSaturatedInto, Permill};
+// --- darwinia ---
+use crate::*;
+use darwinia_fee_market::{Config, RingBalance, Slasher};
+
+frame_support::parameter_types! {
+	pub const FeeMarketPalletId: PalletId = PalletId(*b"da/feemk");
+	pub const TreasuryPalletId: PalletId = PalletId(*b"da/trsry");
+	pub const FeeMarketLockId: LockIdentifier = *b"da/feelf";
+
+	pub const AssignedRelayersNumber: u64 = 3;
+	pub const MinimumRelayFee: Balance = 15 * COIN;
+	pub const CollateralPerOrder: Balance = 100 * COIN;
```

Comment:
Please change `CollateralPerOrder` to `50 * COIN`. I did not find this in HackerFisher's review suggestions. We miss this one. The config in crab should also be updated.